docs/platform/block-storage/block-storage-use-cases/index.md

```diff
@@ -8,58 +8,80 @@ license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
 
 ### Persistent Storage
 
-Block Storage Volumes can be unmounted just as easily as they are mounted, meaning that it's possible to create hot-swappable drives with Block Storage. This is useful if you need to perform the same kind of tasks across a fleet of servers with the same data.
+Block Storage Volumes can be detached from a cloud instance just as easily as they are attached, meaning that it's possible to create hot-swappable drives with Block Storage. This is useful if you need to perform the same kind of tasks across a fleet of instances with the same data.
 
 {{< caution >}}
-While Block Storage is perfectly capable of hot swapping, it is important to follow the set of directions outlined in our [Using Block Storage](/docs/platform/block-storage/how-to-use-block-storage-with-your-linode/#how-to-detach-a-block-storage-volume-from-a-linode) guide, or there is a risk of data loss.
+While the Block Storage service has full support for hot swapping, it is important to follow the detachment instructions outlined in our [Using Block Storage](/docs/platform/block-storage/how-to-use-block-storage-with-your-linode/#how-to-detach-a-block-storage-volume-from-a-linode) guide. If a Volume is not safely detached, there is a risk of data loss for the Volume.
 {{< /caution >}}
 
 ### Container Storage
 
-Containers, like those created with Docker or Kubernetes Pods, can benefit from having some type of persistent storage. This helps to keep the container size down, and makes it easy to maintain data despite the lifecycle of the container or Pod. If you are using Docker, you can use the [Docker Volume Driver for Linode](https://github.com/linode/docker-volume-linode) to create a Docker volume from a Block Storage Volume. Similarly, if you are using Kubernetes you can use the [Container Storage Interface (CSI) Driver for Linode Block Storage](https://github.com/linode/linode-blockstorage-csi-driver) to create a Block Storage Volume backed Persistent Volume Claim.
+Containers, like those created with Docker or inside Kubernetes Pods, can benefit from having some type of persistent storage. This helps to keep a container's size down and makes it easy to maintain data outside of the normal lifecycle of the container or Pod.
+
+If you are using Docker, you can use the [Docker Volume Driver for Linode](https://github.com/linode/docker-volume-linode) to create a Docker volume from a Block Storage Volume. Similarly, if you are using Kubernetes you can use the [Container Storage Interface (CSI) Driver for Linode Block Storage](https://github.com/linode/linode-blockstorage-csi-driver) to create a Persistent Volume Claim that's backed by a Block Storage Volume.
```

Comment:
changed last part to `a Persistent Volume Claim that's backed by a Block Storage Volume`

was going to add a hyphen to make it `Block Storage Volume-backed Persistent Volume Claim` but it felt like the procession of product names was too rapid-fire